### DISCUSSION TITLE
Using SPDX reports?

### DISCUSSION THREAD

**Discussion by rettichschnidi** [fossology/fossology#2455] [General]
I do have a SPDX report provided by an external supplier and would like to make use of it.

Files I got:
 - NAME.spdx.json
 - NAME.spdx.index.json
 - A files files folder with .spdx.json files for every package

I have not found a way to make use of this information, so that I do not have to upload and re-analyze every package source (there are 2500+).

Is this possible with the current state of FOSSology?

**Comment by GMishx** [comment]
Hello @rettichschnidi , thanks for reaching out.

FOSSology currently only understands RDF file format, not the JSON. You can convert the JSON files to "RDF/XML" format using SPDX online tools: https://tools.spdx.org/app/convert/

Then, you can import the files to FOSSology using Report Import agent. You can find documentation about it here: https://github.com/fossology/fossology/wiki/Report-Import

**Comment by rettichschnidi** [comment]
Thanks!
